src/util-debug.c

```diff
@@ -224,6 +224,14 @@ static int SCLogMessageJSON(struct timeval *tval, char *buffer, size_t buffer_si
 
     json_object_set_new(js, "event_type", json_string("engine"));
 
+    const char *s = NULL;
+    s = SCMapEnumValueToName(log_level, sc_log_level_map);
```

Comment:
the variable declaration could be on this line as well, makes the code a bit more compact

Reply:
Corrected in #3558.